lib/src/container/store.rs

```diff
@@ -204,11 +204,10 @@ pub struct PreparedImport {
 }
 
 impl PreparedImport {
-    /// Iterate over all layers; the ostree split object layers, the commit layer, and any non-ostree layers.
+    /// Iterate over all layers; the commit layer, the ostree split object layers, and any non-ostree layers.
     pub fn all_layers(&self) -> impl Iterator<Item = &ManifestLayerState> {
-        self.ostree_layers
-            .iter()
-            .chain(std::iter::once(&self.ostree_commit_layer))
+        std::iter::once(&self.ostree_commit_layer)
```

Comment:
Why this change?

Reply:
This is fixing a bug that existed before. Essentially  the way we are storing the oci images is "first is ostree_commit_layer, then layers with packages", but when we were doing `layers_to_fetch` the order in `all_layers` was reversed causing wrong asserts to be tested here https://github.com/ostreedev/ostree-rs-ext/blob/main/lib/tests/it/main.rs#L728

Reply:
Hmmm.  Can you break that out as a separate change in a new PR?  The tests are passing with the current code, right?  So I assume we need a change to the tests to show the failure or so?

Reply:
Followup here:
https://github.com/ostreedev/ostree-rs-ext/issues/480
https://github.com/ostreedev/ostree-rs-ext/pull/481